accounts/forms.py

```diff
@@ -327,3 +319,18 @@ def __init__(self, *args, **kwargs):
                 }
         super(DeleteUserForm, self).__init__(*args, **kwargs)
 
+
+class EmailSettingsForm(forms.Form):
+    enabled_stream_emails = forms.BooleanField(
```

Comment:
What if we add a field to EmailPreferenceType with the default value? e.g it would be false for StreamEmails and true for the others.